[Python] 1/n Serialize Setup

LastMileNewGrad opened the pull request:
[Python] 1/n Serialize Setup

Implemented Serialize in ConfigRuntime which calls ModelParser
`serialize`

---
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/lastmile-ai/aiconfig/pull/32).
* #33
* __->__ #32

saqadri commented on python/src/aiconfig/Config.py:
```diff
@@ -113,6 +113,25 @@ def load_from_workbook(cls, workbook_id: str) -> "AIConfigRuntime":
+        if not model_parser:
+            raise ValueError(f"Unable to serialize data: `{data}`\n Model Parser for model {model_name} does not exist.")
+        
+        prompts = model_parser.serialize("", data, self, params)
```
What is the empty string for? If it's intentional please comment the variable name so it's easy to know

saqadri reviewed (approved):
Approving to unblock -- let's pass in prompt_name to the new serialize function so we don't pass empty string as the prompt name to model_parser.serialize

LastMileNewGrad commented:
Updated ModelParser and ConfigRuntime

> Approving to unblock -- let's pass in prompt_name to the new serialize function so we don't pass empty string as the prompt name to model_parser.serialize

Updated ModelParser and ConfigRuntime
